packages/modular-scripts/src/react-scripts/config/parts/pluginConfig.ts

```diff
@@ -106,7 +84,6 @@ module.exports = function createPluginConfig({
         new ForkTsCheckerWebpackPlugin({
           async: isEnvDevelopment,
           typescript: {
-            async: isEnvDevelopment,
```

Comment:
webpack config type claims async isn't a property of typescript {}

Reply:
Allowed properties of typescript:
```
    memoryLimit?: number;
    configFile?: string;
    configOverwrite?: TypeScriptConfigOverwrite;
    context?: string;
    build?: boolean;
    mode?: 'readonly' | 'write-tsbuildinfo' | 'write-dts' | 'write-references';
    diagnosticOptions?: Partial<TypeScriptDiagnosticsOptions>;
    extensions?: {
        vue?: TypeScriptVueExtensionOptions;
    };
    profile?: boolean;
    typescriptPath?: string;
```

I think we can put this down as a mistake until now uncaught because of lack of typings